### PR TITLE
add content about commit rule

### PR DESCRIPTION
**Pull request by chokobole**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -102,6 +102,13 @@ Exceptionally, commit type `docs` may omit scope. e.g, docs: update contracts/RE
 If the change affects more than one scope, the commit scope may be omitted.
 e.g, feat: add Colosseum contract
 
+### Commit Rule
+
+There are the following rules for commit messages:
+
+- The commit title should always start with an imperative verb.
```
Comment (fakedev9999) [inline CONTRIBUTING.md:215]: How about adding the rules for the Pull Requests as well in a similar manner? There might be other more rules for Pull Requests, but at least we need a rule with the PR title.

**Review by Insun35 — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ We follow [google coding style](http://google.github.io/styleguide/).
 
 Before Pull Request, you have to check followings first.
```
Comment (ashjeong) [inline CONTRIBUTING.md]:
```Suggestion
Before creating a Pull Request, you must first check the following:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ We follow [google coding style](http://google.github.io/styleguide/).
...
 
-- Whether there is typo in the subject and body of your commit.
+- Whether there are typos in the subject and body of your commit.
 - Whether the subject of your commit explains well about your changes.
```
Comment (ashjeong) [inline CONTRIBUTING.md]:
``` Suggestion
- Whether the subject of your commit explains your changes well.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (ashjeong) [reply CONTRIBUTING.md:215]: Parts of commit rules and PR rules would overlap -> aka like "If the change affects more than one scope, the commit scope may be omitted. e.g, feat: add Colosseum contract" -> but I agree that separating the two categories in a standardized way would be better.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -102,6 +102,13 @@ Exceptionally, commit type `docs` may omit scope. e.g, docs: update contracts/RE
 If the change affects more than one scope, the commit scope may be omitted.
 e.g, feat: add Colosseum contract
 
+### Commit Rule
+
+There are the following rules for commit messages:
```
Comment (ashjeong) [inline CONTRIBUTING.md]:
``` Suggestion
Here are the following rules for commit messages:
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -102,6 +102,13 @@ Exceptionally, commit type `docs` may omit scope. e.g, docs: update contracts/RE
 If the change affects more than one scope, the commit scope may be omitted.
 e.g, feat: add Colosseum contract
 
+### Commit Rule
```
Comment (ashjeong) [inline CONTRIBUTING.md]:
``` Suggestion
### Commit Rules
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (ashjeong) [reply CONTRIBUTING.md]: or "accurately" might be better

File: CONTRIBUTING.md (hunk 1 above)
Comment (chokobole) [reply CONTRIBUTING.md:215]: Since this PR is pending so long, I think I want to ask @ashjeong to continue on it.

**Pull request merged**